troubleshooting/common_issues/index.rst

```diff
@@ -60,3 +60,20 @@ You can revert this operation by running the following commands:
     rm -rf ./app/cache/*
     php app/console pim:install:asset --env=prod
     php app/console assets:install --symlink web
+    
+I cannot export my products "An exception occurred during the export"
+--------------------------------------------------------------------
+
+If you have **thousands of products to export with the native csv or xslx connector** and if the job finishes or fails with the error: **"An exception occured during the export"**, first check the prod.log to have more information about the error.
+
+If the system mentions an error like "CRITICAL: Fatal Error: Allowed memory size of bytes exhausted (tried to allocate XXXXXXXX bytes)" **it means that there is a memory leak and it might be linked to the media archiving**. We have noticed that exporting more than 1 Gb of medias could lead to a memory leak :doc:`/reference/scalability_guide/more_than_1GB_of_product_media_to_export.html`
```

Comment:
Also this is directly related to the `memory_limit` in your _/etc/php5/cli/php.ini_ file. Another error could be "Out of memory" which is when you have set your `memory_limit` higher that what your system can actually support in terms of RAM.  It can occurs also if you set `memory_limit = -1` for this reason.